src/search/landmarks/landmark_factory_merged.h

```diff
@@ -9,14 +9,17 @@ namespace landmarks {
 class LandmarkFactoryMerged : public LandmarkFactory {
     std::vector<std::shared_ptr<LandmarkFactory>> landmark_factories;
 
-    std::vector<std::shared_ptr<LandmarkGraph>> generate_landmark_graphs_of_subfactories(
+    std::vector<std::shared_ptr<LandmarkGraph>>
+    generate_landmark_graphs_of_subfactories(
         const std::shared_ptr<AbstractTask> &task);
-    void add_atomic_landmarks(
-        const std::vector<std::shared_ptr<LandmarkGraph>> &landmark_graphs) const;
+    void add_atomic_landmarks(const std::vector<std::shared_ptr<LandmarkGraph>>
+                                  &landmark_graphs) const;
```

Comment:
I would prefer the way it was before in this case.

Reply:
Agreed, an unfortunate victim of being strict about the column limit.

Reply:
It would break after the opening parenthesis if everything after it fit on a single line, but with the `const` and all it ends up 2 characters too long in this case. With a slightly shorter variable name it would look like this:
```c++
    void add_atomic_landmarks(
        const std::vector<std::shared_ptr<LandmarkGraph>> &landmark_grap) const;
```
Breaking at the `const` would help too, but it doesn't do so and I can't find a way to enable something like that.

Reply:
And I don't think that I would like it better if it broke at either one of the `const`s.